### DISCUSSION TITLE
Confusion about the v3.7.1 and v4 releases

### DISCUSSION THREAD

**Discussion by lovromazgon** [golangci/golangci-lint-action#967] [Q&A] [answered]
On Feb 5, 2024 there were 3 tags pushed to the action repository:

- [`v3.7.1`](https://github.com/golangci/golangci-lint-action/releases/tag/v3.7.1)
- [`v4`](https://github.com/golangci/golangci-lint-action/releases/tag/v4)
- [`v4.0.0`](https://github.com/golangci/golangci-lint-action/releases/tag/v4.0.0)

I'm confused because no release was created for any of these tags. Also, two different major versions are now pinned to the same commit, which seems strange.

Was this a mistake, or was this done on purpose?

**Comment by ldez** [comment, marked as answer]
https://github.com/golangci/golangci-lint-action/issues/953
https://github.com/golangci/golangci-lint-action/releases/tag/v4.0.0
